### DISCUSSION TITLE
Fail load JS after build

### DISCUSSION THREAD

**Discussion by kantin13** [nuxt/nuxt#17849] [Questions]
### Environment

- Operating System: `Darwin`
- Node Version:     `v16.13.0`
- Nuxt Version:     `3.0.0-27447229.20f3171`
- Package Manager:  `npm@8.1.0`
- Bundler:          `Vite`
- User Config:      `ssr`, `nitro`, `meta`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

https://github.com/kantin13/failLoadJsAfterBuild

### Describe the bug

Hello,
I am migrating my project to Nuxt V3 and I have an error that I cannot solve. I want to import a js file in the head and call a function when the page is loaded. Any function when I am in npm run dev but when I build my function is not recognized anymore, any idea?

<img width="925" alt="image" src="https://user-images.githubusercontent.com/25583086/157890488-c0943607-eabf-4af8-b54e-6a9e70848520.png">

<img width="925" alt="image" src="https://user-images.githubusercontent.com/25583086/157890448-1707a575-8c83-40ea-9960-e789b22af5d4.png">

and after build :
<img width="629" alt="image" src="https://user-images.githubusercontent.com/25583086/157890564-1715d8bb-e087-4d30-b172-0bdd16d78cff.png">

**Comment by kantin13** [comment]
Does anyone have an idea?

**Comment by danielroe** [comment]
This is a timing issue. With `ssr: false` the rendered HTML doesn't include the script, so it is loaded in after Nuxt loads. But your mounted hook expects it already to be loaded by the browser.
